org-formation/090-systems-manager/_tasks.yaml

```diff
@@ -61,6 +61,18 @@ PatchMembers:
   Parameters:
     ManagementAccountNumber: !Ref accountId
 
+PatchPolicy:
```

Comment:
how about naming it `QuickSetupPatchPolicy`?   

Does this conflict with the existing patch setup?  Is the idea to install this then remove the old setup or do we need to remove the old setup first?

Reply:
No, they don't conflict, we can run both at the same time. The idea is to add this new process, then remove the old setup afterwards.